self_study_task1.py

```diff
@@ -0,0 +1,97 @@
+import random
```

Comment:
This is a great self study Nina, i like how your functions are documented, and they do just what the name suggests. This is really awesome